version.cake

```diff
@@ -265,27 +265,27 @@ void UpdateWrapperSdkVersion(string newVersion)
 
 void UpdateNewProjSdkVersion(string newVersion, string newFileVersion)
 {
-    var csprojFiles = GetFiles("SDK/**/*.csproj");
```

Comment:
Can you explain what's the changes in this file?

Reply:
To pick up version change on wpf .net core test app, was working only on AssemblyInfo.cs before for apps.